week3/src/components/Header.jsx

```diff
@@ -0,0 +1,72 @@
+// styled API를 사용하여 스타일링
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import "./SingleCard.css";
+
+const Header = ({ counter, length }) => {
+  const [animate, setAnimate] = useState(false);
+  useEffect(() => {
+    if (counter != 0) {
+      setAnimate(true);
+      setTimeout(() => setAnimate(false), 1000);
+    }
+  }, [counter]);
+  return (
+    <>
+      <ChokiPokiHeader>
+        <strong> 💗 쵸키랑 푸키를 맞춰주세요! 💗</strong>
+        <AnswerCounter className={animate ? "scale" : ""}>
```

Comment:
animate 이면 scale 효과를 주려고 넣은건강?! 그러면 그냥 요런식으로 props로 넘겨줘서 조건부로 스타일 적용할 수 있지 않으까??!

```suggestion
        <AnswerCounter animate={animate}>
```